ts/backend/src/app/features/controller/results.controller.mts

```diff
@@ -550,7 +555,7 @@ export class ResultsController extends Controller {
    *                                  format: uuid
    *                                  description: ID of best submission.
    */
-  getCampaignItem: GetHandler<'/results/campaign-items/:benchmark_id'> = async (req, res) => {
+  getCampaignItemOverview: GetHandler<'/results/campaign-items/:benchmark_id'> = async (req, res) => {
```

Comment:
should this not allow for allow for a list of campaign-items to have all GET endpoints `/FOOs/{id}` consistently returning list as the endpoint as dev.002/dev.003 suggests?